sw/device/tests/crypto/cryptotest/json/ecdsa_commands.h

```diff
@@ -0,0 +1,62 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_JSON_ECDSA_COMMANDS_H_
+#define OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_JSON_ECDSA_COMMANDS_H_
+#include "sw/device/lib/ujson/ujson_derive.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define ECDSA_CMD_MAX_MESSAGE_BYTES 64
+#define ECDSA_CMD_MAX_SIGNATURE_SCALAR_BYTES 64
+#define ECDSA_CMD_MAX_COORDINATE_BYTES 64
+
+// clang-format off
+
+#define ECDSA_OPERATION(_, value) \
+    value(_, Verify)
+UJSON_SERDE_ENUM(CryptotestEcdsaOperation, cryptotest_ecdsa_operation_t, ECDSA_OPERATION);
+
+#define ECDSA_HASH_ALG(_, value) \
+    value(_, Sha256) \
+    value(_, Sha384) \
+    value(_, Sha512) \
+    value(_, Sha3_256) \
+    value(_, Sha3_512)
```

Comment:
Missing SHA3-384?
```suggestion
    value(_, Sha3_256) \
    value(_, Sha3_384) \
    value(_, Sha3_512)
```

Reply:
I originally left it out because we don't have any test vectors, but we probably should include it just so any future test vectors for it will be plug-and-play.